actors/evm/src/interpreter/instructions/memory.rs

```diff
@@ -60,7 +46,7 @@ pub fn get_memory_region(
 
     let current_size = mem.len();
     if new_size as usize > current_size {
-        grow_memory(mem, new_size as usize);
+        mem.grow(new_size as usize);
```

Comment:
Can we get rid of the size check here?

Reply:
(and rename `grow` to something like `ensure_capacity` or `ensure_size`, or something)

Reply:
Scratch the second case.